Pipfile

```diff
@@ -28,4 +25,4 @@ gene-normalizer = "~=0.1.36"
 pyliftover = "*"
```

Comment:
I think you can remove the pyliftover dependency now since it's just used in cool-seq-tool?